recipes/application_python.rb

```diff
@@ -95,6 +95,7 @@
 if node.deep_fetch(stackname, 'code-deployment', 'enabled')
   node[stackname][node[stackname]['webserver']]['sites'].each do |port, sites|
     sites.each do |site_name, site_opts|
+      next if site_opts['repository'] == ''
```

Comment:
maybe we don't need to default to en empty string and test an empty string, we could just do a `if site_opts['repository']` ?

Except this at least it fixes those 2 attributes for now. We can look later if this approach default_unless approach breaks something.